### DISCUSSION TITLE
keyball61でのドラッグ＆ドロップについて

### DISCUSSION THREAD

**Discussion by totorokids** [Yowkees/keyball#149] [Q&A]
keyballを初めて自作したのですが、ドラッグ＆ドロップの方法が分からず質問させていただきました。
Macbook Proで使用しています。

**Comment by totorokids** [comment]
解決しました。大変失礼しました。

**Comment by Yowkees** [comment]
ご連絡が遅くなり申し訳ありませんでした。
ご自身で解決できたとのことで良かったです。
末永くご愛用いただければ幸いです！
